cluster/gke.go

```diff
@@ -1961,7 +1961,7 @@ func (c *GKECluster) IsReady() (bool, error) {
 		return false, err
 	}
 
-	c.log.Debug("Get gke cluster with name %s", c.model.Cluster.Name)
+	c.log.Debugf("Get gke cluster with name %s", c.model.Cluster.Name)
```

Comment:
```suggestion
	c.log.Debug("Get gke cluster with name")
```

either cluster id or cluster name field is already set on `c.log`